utils/logger.go

```diff
@@ -0,0 +1,109 @@
+// this is a new logger interface for mattermost
+
+package utils
```

Comment:
That's fine with me. When I got started, I asked @coreyhulen where to put it, and he suggested it belong in utils for the time being.